config/nginx/templates/cms.conf.templ

```diff
@@ -1,7 +1,20 @@
+{% if ACTIVATE_HTTPS %}
 server {
-  listen 80;
+    server_name {{ CMS_HOST }};
+    listen 80;
+    return 301 https://$server_name$request_uri;
+}
+{% endif %}
+
+server {
+  listen {{ "443 ssl" if ACTIVATE_HTTPS else "80" }};
   server_name studio.localhost {{ CMS_HOST }};
-  
+
+  {% if ACTIVATE_HTTPS %}
+  ssl_certificate /etc/letsencrypt/live/{{ CMS_HOST }}/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/{{ CMS_HOST }}/privkey.pem;
```

Comment:
On both those lines, we should have `LMS_HOST` instead of `CMS_HOST`. This is because the certbot command generates a single certificate for the entire platform.